setup.py

```diff
@@ -13,13 +13,13 @@
     packages=['aubreylib'],
     install_requires=[
         'lxml>=3.3.3',
-        'pyuntl>=1.0.1',
```

Comment:
Why are we removing this from here?

Reply:
I tried using the pyuntl dependency from install_requires but it wasn't getting installed properly. So I tried to move it from here to requirements and it worked!